cmd/main.go

```diff
@@ -118,6 +118,9 @@ var Options struct {
 	DeregisterWorkerInterval    time.Duration `envconfig:"DEREGISTER_WORKER_INTERVAL" default:"1h"`
 	EnableDeletedUnregisteredGC bool          `envconfig:"ENABLE_DELETE_UNREGISTER_GC" default:"true"`
 	EnableDeregisterInactiveGC  bool          `envconfig:"ENABLE_DEREGISTER_INACTIVE_GC" default:"true"`
+	ServeHTTPS                  bool          `envconfig:"SERVE_HTTPS" default:"false"`
+	HttpsKeyFile                string        `envconfig:"HTTPS_KEY_FILE" default:""`
```

Comment:
Please pick one capitalization method for HTTPS in the new fields  (Https vs. HTTPS) for consistency.

Reply:
:facepalm:

Reply:
Also this is in a commit from https://github.com/openshift/assisted-service/pull/1495. Fixed it there.